server/services/analyticsServiceTypes.ts

```diff
@@ -63,10 +62,7 @@ export type BehaviourEntriesByLocation = {
 export type PrisonersWithEntriesByLocation = {
   location: string
   href?: string
-  prisonersWithPositive: number
-  prisonersWithNegative: number
-  prisonersWithBoth: number
-  prisonersWithNeither: number
+  values: [number, number, number, number]
```

Comment:
As things are getting more generic I think they're starting to be more abstract and potentially confusing. 

In this specific case for example, the numbers had a name and you could tell what was their meaning, but now it's just an array of numbers.

I'm wondering if add some comments would be a good idea, e.g.

```
// ...
// [prisonersWithPositive, prisonersWithNegative, prisonersWithBoth, prisonersWithBoth]
values: [number, number, number, number]
```

but maybe there is a better way of retaining the semantics.

Reply:
the reason it's all getting more generic was that the templates were going to explode soon: there will be some 30 graphs. it'd be a real pain (end error prone!) to synchronise that many templates if each one used different variables and subtle variation in logic.

i'm ok with adding extra comments tho